images.CI/shebang-linter.ps1

```diff
@@ -0,0 +1,34 @@
+$ErrorActionPreference = "Stop"
+
+function Validate-Scripts {
+    Param (
+        [Parameter(Mandatory=$true)]
+        [string[]]$Path,
+        [Parameter(Mandatory=$true)]
+        [string]$ExpectedShebang
+    )
+    $ScriptWithoutShebangLine = @()
+    Get-ChildItem $path -Recurse -File -Filter "*.sh" | ForEach-Object {
+        $shebangLine = Get-Content -Path $($_.FullName) | Select-Object -First 1
+        if ($shebangLine -eq $ExpectedShebang) {
+            Write-Host "Pattern '$ExpectedShebang' found in '$($_.FullName)'"
+        }
+        else {
+            Write-Host "Pattern '$ExpectedShebang' not found in '$($_.FullName)'"
```

Comment:
Do we need to duplicate the same message at 17 and 31 lines?

Reply:
I believe we do need since warning messages will be highlighted and it's easy to find all failed script at the end of the log output.